src/main/java/org/openhab/binding/zigbee/converter/ZigBeeConverterColorTemperature.java

```diff
@@ -97,9 +99,24 @@ public void run() {
 
     @Override
     public Channel getChannel(ThingUID thingUID, ZigBeeEndpoint endpoint) {
-        if (endpoint.getInputCluster(ZclColorControlCluster.CLUSTER_ID) == null) {
+        clusterColorControl = (ZclColorControlCluster) endpoint.getInputCluster(ZclColorControlCluster.CLUSTER_ID);
+        if (clusterColorControl == null) {
             return null;
         }
+
+        try {
+            if(!clusterColorControl.discoverAttributes(false).get()) {
+                logger.warn("{}: Cannot determine whether device endpoint supports color temperature. Assuming it does", endpoint.getEndpointAddress());
+            }
+            else if(!clusterColorControl.getSupportedAttributes().contains(ZclColorControlCluster.ATTR_COLORTEMPERATURE)) {
+                logger.debug("{}: Device endpoint does not support color temperature (probably only RGB color)", endpoint.getEndpointAddress());
```

Comment:
I'm not sure we really need this log entry as we dont normally log channels that aren't supported?

Reply:
Useful for me to debug while coding, but I can remove it for the PR